makefile

```diff
@@ -89,12 +93,8 @@ $(buildDir)/evergreen_ui_server:scripts/evergreen_ui_server
 
 
 # start rules for building services and clients
-ifeq ($(OS),Windows_NT)
-cli:$(clientBuildDir)/$(goos)_$(goarch)/evergreen.exe
-else
-cli:$(clientBuildDir)/$(goos)_$(goarch)/evergreen
-endif
-clis:$(clientBinaries)
+cli:$(clientBinary)
+clis:$(if $(DISABLE_CROSS_COMPILE),$(clientBinary),$(clientBinaries))
```

Comment:
This is approach is fine, but my goal with #79 was to provide a dist target, which this does not do.